content/python/concepts/seaborn/seaborn.md

```diff
@@ -0,0 +1,12 @@
+---
+Title: 'Python:Seaborn'
+Description: 'Seaborn is a popular open-source Python library, used for visualizing the explorative statistical plots of data.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/python'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+[Seaborn](https://seaborn.pydata.org/) is a popular open-source Python library, used for visualizing the explorative statistical plots of data. It is a basic libary used to extract various information that can provide an understanding of your dataset. To use Seaborn's objects and methods, it will need to be installed first. There are instructions on how to do so on the [Seaborn website](https://seaborn.pydata.org/installing.html).
```

Comment:
```suggestion
[Seaborn](https://seaborn.pydata.org/) is a popular open-source Python library, used for visualizing the explorative statistical plots of data. It is a basic libary used to extract various information that can provide an understanding of a dataset. To use Seaborn's objects and methods, it will need to be installed first. There are instructions on how to do so on the [Seaborn website](https://seaborn.pydata.org/installing.html).
```